.github/workflows/ci.yml

```diff
@@ -39,6 +39,34 @@ jobs:
       - name: Run tests
         run: |
           cargo test --all-targets --all-features
+  
+  run-fuzzer:
+    name: Run fuzzer
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Install nightly toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: nightly
+          components: rust-src
+
+      - name: Use MSRV-specific .cargo/config.toml
+        run: |
+          mv .cargo/config.toml.msrv .cargo/config.toml
+
+      - name: Install cargo-fuzz
+        uses: baptiste0928/cargo-install@v3
+        with:
+          crate: cargo-fuzz
+          locked: false
+
+      - name: Run stream fuzzer for a limited time
+        run: |
+          RUST_BACKTRACE=1 cargo fuzz run stream -- -max_total_time=60
```

Comment:
I think it should be fine to run for longer than a minute, the powerset CI already runs for a long time anyways

Reply:
Since powerset takes ~15m, I raised fuzzing to 15m